src/Umbraco.Web.UI.Client/package.json

```
@@ -149,7 +149,7 @@
 		"backoffice:test:e2e": "npx playwright test",
 		"build-storybook": "npm run wc-analyze && storybook build",
 		"build:for:cms": "npm run build && npm run build:workspaces && npm run generate:manifest && npm run package:validate && node ./devops/build/copy-to-cms.js",
-		"build:for:npm": "npm run build -- --declaration && npm run generate:manifest && npm run package:validate",
+		"build:for:npm": "npm run build -- --declaration && npm run generate:manifest && npm run package:validate && npm run generate:jsonschema:dist && npm run wc-analyze",
```

We don't need to add the generation here as that is already being done in the **prepack** step:

```suggestion
		"build:for:npm": "npm run build -- --declaration && npm run generate:manifest && npm run package:validate",
```